cmd/commands/runtime.go

```diff
@@ -553,20 +558,20 @@ func getComponents(rt *runtime.Runtime, opts *RuntimeInstallOptions) []string {
 	return componentNames
 }
 
-func createRuntimeOnPlatform(ctx context.Context, opts *model.RuntimeInstallationArgs) (string, string, error) {
+func createRuntimeOnPlatform(ctx context.Context, opts *model.RuntimeInstallationArgs) (string, string, string, error) {
 	runtimeCreationResponse, err := cfConfig.NewClient().V2().Runtime().Create(ctx, opts)
 	if err != nil {
-		return "", "", fmt.Errorf("failed to create a new runtime: %s. Error: %w", opts.RuntimeName, err)
+		return "", "", "", fmt.Errorf("failed to create a new runtime: %s. Error: %w", opts.RuntimeName, err)
 	}
 
 	const IV_LENGTH = 16
 	iv := make([]byte, IV_LENGTH)
 	_, err = io.ReadFull(rand.Reader, iv)
 	if err != nil {
-		return "", "", fmt.Errorf("failed to create an initialization vector: %s. Error: %w", opts.RuntimeName, err)
+		return "", "", "", fmt.Errorf("failed to create an initialization vector: %s. Error: %w", opts.RuntimeName, err)
 	}
 
-	return runtimeCreationResponse.NewAccessToken, hex.EncodeToString(iv), nil
+	return runtimeCreationResponse.NewAccessToken, hex.EncodeToString(iv), runtimeCreationResponse.SharedConfigRepo, nil
```

Comment:
Where can a user control and set his shared config repo? how is it determined what is the shared config repo?